Modules/Scripted/DICOMLib/DICOMBrowser.py

```diff
@@ -362,8 +362,7 @@ def organizeLoadables(self):
         if loadable.confidence > highestConfidenceValue:
           highestConfidenceValue = loadable.confidence
       for loadable in loadablesBySeries[series]:
-        if loadable.confidence < highestConfidenceValue:
-          loadable.selected = False
+        loadable.selected = loadable.confidence == highestConfidenceValue
```

Comment:
We had to add this mechanism (keep only a single loadable) for a good reason but I forgot what was that. To avoid this in the future, please add a comment here that explains what the current mechanism does and why (basically what you described in the commit comment).

Reply:
Okay, let's keep an eye out for any cases where this leads to trouble.

Reply:
Yes, it could happen that too many subseries get loaded, but I'm not a fan of an arbitrary cutoff.  I hope most of those cases would become sequences, or even better get caught by some acquisition-specific code like we had in the Longitudinal PETCT extension.